.trivyignore

```diff
@@ -4,6 +4,10 @@ GHSA-fr2g-9hjm-wr23
 CVE-2023-39325
 CVE-2023-3978
 CVE-2023-44487
+GHSA-jq35-85cj-fj4p
+
+#telegraf HIGH
+GHSA-m425-mq94-257g
```

Comment:
Dont we need to address for fedramp compliance?

Reply:
Will add telegraf update in this PR